apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -718,9 +817,16 @@ def grade(element_html, data):
         if final_score < 1:
             if feedback_type == "none":
                 feedback = ""
-            elif feedback_type == "first-wrong":
+            elif feedback_type.startswith("first-wrong"):
                 if first_wrong == -1:
                     feedback = FIRST_WRONG_FEEDBACK["incomplete"]
+                elif (
+                    feedback_type.endswith("verbose")
```

Comment:
Isn't there only one verbose feedback type? Should explicitly check for it in that case.

Reply:
I fixed all these when I merged with the new typechecking.